test/cli/test/bail-race.test.ts

```diff
@@ -0,0 +1,44 @@
+import { resolve } from 'pathe'
+import { expect, test } from 'vitest'
+import { createVitest } from 'vitest/node'
```

Comment:
We have `runVitest` in `../test-utils` that you can use instead. It sets all the flags (maxWorkers, watch) automatically